boards/PDM/Inc/Io/Io_Efuse.h

```diff
@@ -0,0 +1,257 @@
+/**
+ * @brief Driver for interfacing with the 22XS4200 E-Fuse via SPI
+ * @Settings Efuse max SPI transfer rate: 8MHz
+ *           SPI Clock Polarity: 0 (SCK low-level idle state)
+ *           SPI Clock Phase: 2nd edge (slave samples MOSI on SCK falling edge)
+ *           Slave Select: Active Low (Must be toggled between SPI messages)
+ */
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "main.h"
+#include "App_SharedExitCode.h"
+
+struct Efuse;
+
+// Enumeration for the status types
+enum Efuse_Status
+{
+    StatusType_NoFault = (0 << 0), // No fault
+    StatusType_OUT0    = (1 << 0), // Channel 0 Output state (ON/OFF)
```

Comment:
@markomarkusi I'm assuming StatusType_OUT0 mean that the output state is off and StatusType_OUT1 is on? I can't tell from the comments

Reply:
Yes I make those more clear. Should be more like `StatusType_Output0Enabled` or `StatusType_Channel0Enabled`

Reply:
Previously they were the names  of the register contents
![image](https://user-images.githubusercontent.com/25499626/87832392-c65f1580-c83a-11ea-9163-3a1ce8df3f0d.png)
Changed them to longer names if you like that more.